internal/provider/datasource_rediscloud_subscription_peerings.go

```diff
@@ -82,6 +82,26 @@ func dataSourceRedisCloudSubscriptionPeerings() *schema.Resource {
 							Type:        schema.TypeString,
 							Computed:    true,
 						},
+						"aws_peering_id": {
+							Description: "Identifier of the AWS cloud peering",
+							Type:        schema.TypeString,
+							Computed:    true,
+						},
+						"redis_project_id": {
```

Comment:
should we make these new fields `gcp_redis_project_id`, `gcp_redis_network_name` and `gcp_peering_id` so that everything related to GCP start with `gcp` and it's more consistent with the two other ones: `gcp_network_name` and `gcp_project_id`? 
Also `gcp_peering_id` would be more aligned to AWS's counterpart `aws_peering_id`

Reply:
These have been updated to have a prefix of `gcp_redis`.  This feels clear.

Reply:
I would still rename the `cloud_peering_id` to `gcp_peering_id`

Reply:
Renamed cloud_peering_id to gcp_peering_id.